### PR TITLE
Support RAW write operations

### PR DESCRIPTION
**Pull request by waldbaer**
Hello @coolchip,

this is the support for https://github.com/coolchip/luxtronik2/pull/20 in the node-red wrapper.
Please let me know if something could be improved.

Best regards
Sebastian

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -28,16 +28,29 @@ module.exports = function (RED) {
...
-            pump.write(parameterName, realValue, function (err, data) {
-                if (err) {
-                    msg.payload = err;
-                    return node.send(msg);
-                }
-                msg.payload = data;
-                node.send(msg);
-            });
+            var parameterValue = msg.payload;
+            var rawParameterNumber = msg.raw_parameter;
```
Comment (coolchip) [inline index.js]: rawParameter instead of rawParameterNumber?

Comment (waldbaer) [reply index.js]: Makes sense.
I renamed it.

Comment (waldbaer) [reply index.js]: @coolchip Did you already check the little patch?

**Comment by coolchip** [comment]
Sorry for letting you wait. Will publish a version soon.

**Comment by coolchip** [comment]
Done

**Pull request merged**